packages/terra-core-docs/CHANGELOG.md

```diff
@@ -28,6 +28,28 @@
   * Added docs for `AnchorButton`.
   * Added test for `AnchorButton`.
 
+* Fixed
+  * Updated broken links in terra-form-select and action-footer.
+  * linter fixes.
```

Comment:
Looks like these lines got merged in error since the current [changlog](https://github.com/cerner/terra-core/blob/main/packages/terra-core-docs/CHANGELOG.md) in main doesn't have this.